### ISSUE TITLE
Migrate to event page

### ISSUE THREAD

**Issue by kav2k** [knyar/theoldreader-chrome#32]
Work started by @Vinne2, currently in branch `event-page`, is aimed at replacing the persistent background page with an event one.

In principle, this is doable, but there are many effects to consider.
- [x] Finish transition of `getCountersFromHTTP()` to `chrome.alarms`
- [x] Test optimistic retry (#22) with 1-minute granularity
- [ ] Consider effects of transition for `storage.js`, i.e. state
- [ ] Transition `saveToStorage()` to `chrome.alarms` (see `syncRetryTimeout`)
- [ ] Consider top-level code in `background.js`
- [ ] Consider ALL global state in `functions.js`

This is a non-trivial task, all in all, even not considering impact on #22.

**Comment by Vinne2** [comment]
Any progress on that here?

Work must've pulled me away from this long ago and I totally forgot about the case. Now I've been checking my extensions and have just started filing issues about moving them to be persistent:false and came back to this. I fell like in a timeloop…

**Comment by kav2k** [comment]
Nope, no progress. I haven't touched this extension in years (because it works as-is).

**Comment by Vinne2** [comment]
I'd like to make a pull request to the branch event-page here for you to see my work, but the branch is 4 years behind the master. Should you update the event-page branch from master or should I make a pull request anyway? See the problem: https://github.com/knyar/theoldreader-chrome/compare/event-page...Vinne2:event-page?expand=1 The diff gets cluttered with all the 4-years' commits.

**Comment by kav2k** [comment]
Disregard the `event-page` branch here (I'll delete it), PR off the master.
